content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -109,21 +109,21 @@ average and tail latencies. The actual tail latency depends on the traffic patte
 ### Latency for Istio {{< istio_release_name >}}
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2), the two proxies add about 2.8 ms and 2.7 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2), the two proxies add about 3.1 ms to the 90th and 99th percentile latency, over the baseline data plane latency.
```

Comment:
same latency for 90th and 99?

Reply:
one is 3.12 one is 3.13, not sure if we need make it down to 2 bits floating number (i just followed the old convention).

Reply:
please do use 2 digits.

Reply:
sure